steps/change_code.yml

```diff
@@ -12,7 +12,7 @@ startFlow:
           - text: "Now that you have created the CI/CD pipeline, let's make a change to the code and see the pipeline in action."
           - text: "We will make a small change to the style. Let's change the background color of the calculator results bar."
           - text: "So, :instruction[open the `public/default.css` file and change the **`#result`** CSS selector `background-color` to `#5362e0`]"
-          - text: "Once you're done, open a pull request and you should see your action running. I'll be here to review it."
+          - text: Once you're done, :instruction[create a new branch and open a PR with your changes, and I'll review it].
```

Comment:
didn't we already created a branch?

Reply:
Does it matter if we created one already? Arent we supposed to create a new branch for each PR?

Reply:
@shem8 what do you say?